### ISSUE TITLE
Posibility to show multiple currency-pairs in the menu bar

### ISSUE THREAD

**Issue by bavarianbytes** [geraldoramos/crypto-bar#33]
That would be great. Want to track ETH and BTC.

**Comment by yumitsu** [comment]
I have to agree. I've used an application called TextBar before, but it's not quite suitable for this, because if it something more complex than `df -h` or `uptime -a` - it's all up to you, go and code it. Not that it's a problem, it's just less flexible.

So, yes, multiple exchange pairs in bar would be a great.
